bigbluebutton-html5/imports/ui/stylesheets/mixins/_indicators.scss

```diff
@@ -0,0 +1,41 @@
+@import "/imports/ui/stylesheets/variables/palette";
+@import "/imports/ui/stylesheets/variables/general";
+
+@mixin presenterIndicator() {
```

Comment:
Please use placeholders here, when compiling the function will be duplicated, but a placeholder will generate classes only by adding the differences.

`